card/mod/05_standard/set/all/error.rb

```diff
@@ -92,6 +92,7 @@ def unsupported_view view
     return '' unless card.ok? :create  # should this be moved into ok_view?
 
     opts = { :remote=>true, :class=>"slotter missing-#{ args[:denied_view] || args[:home_view]}" }
+    opts[:path_opts] = { :type=> args[:type] } if args[:type]
```

Comment:
I think this got lost somehow.  The idea was to make it so that the "add whatever" links to missing nested cards set the type correctly.